deploy/helm/sumologic/templates/_helpers.tpl

```diff
@@ -35,8 +35,12 @@ We truncate at 63 chars because some Kubernetes name fields are limited to this
 {{- template "sumologic.fullname" . }}
 {{- end -}}
 
+{{- define "sumologic.labels.app.fluentd" -}}
```

Comment:
This is a duplicate of `sumologic.metadata.name.fluentd` added below

Reply:
`labels.app` are not `sumologic.metadata`